clients/web/src/templates/body/systemConfiguration.pug

```diff
@@ -1,5 +1,13 @@
 .g-body-title System configuration
 form.g-settings-form(role="form")
+  .g-settings-form-container
+    h4 Instance Branding
+    .form-group
+      label(for="g-core-contact-email-address") Contact email address
+      input#g-core-contact-email-address.form-control.input-sm(
+          type="text", value=settings['core.contact_email_address'] || '',
+          placeholder=`Default: ${defaults['core.contact_email_address'] || 'none'}`,
+          title="The contact email address for support.")
```

Comment:
How about `title="The contact email address for users to obtain support with this instance."`? I'm open to even better phrasing, but this is a little more descriptive.